LAYOUT.md

```diff
@@ -99,8 +99,12 @@ The root directory holds the content of the lesson, and must contain:
     this lesson, and is about the filesystem. (Note that we use hyphens
     rather than underscores in filenames.) See "Topics" below.
 
-4.  `install.md`: instructions for installing the software tools used
-    by learners for this lesson.
+4.  `install.md`: instructions for installing the software tools
+    used by learners for this lesson.
+    Setup instructions for tools used by the core lessons are
+    already available at the
+    [workshop template][swc-workshop-template-rendered] page.
```

Comment:
“… are already available…” is pretty optimistic (and installation instructions change with time ;).  How about focusing on the rationale here:

>  Setup instructions for tools used by the core lessons should be [posted to the workshop-template repository][1], which provides a central location for documenting common tools.  Setup instructions for other lessons should be provided in `install.md`, which scales better as the number of non-core lessons increases.

[1]: https://github.com/swcarpentry/workshop-template/pulls
[2]: https://swcarpentry.github.io/workshop-template/